src/proc.h

```diff
@@ -28,6 +28,12 @@ struct mem_map *create_maps(int pid);
 int print_proc_maps(int pid);
 
 /*
+ * simple routine to print process comm for
+ * debugging or advanced error reporting
+ */
+int print_proc_comm(int pid);
```

Comment:
By the way, I think it's better to name it `print_thread_comm()` to keep current naming conventions.